src/mono/wasm/build/WasmApp.targets

```diff
@@ -109,6 +109,8 @@
     <WasmGenerateAppBundle Condition="'$(WasmGenerateAppBundle)' == ''">false</WasmGenerateAppBundle>
     <UseAppHost>false</UseAppHost>
     <TrimMode Condition="'$(TrimMode)' == ''">partial</TrimMode>
+    <_ExtraTrimmerArgs Condition="'$(WasmEnableSIMD)' == 'true'">$(_ExtraTrimmerArgs) --substitutions $(MSBuildThisFileDirectory)/ILLink.Substitutions.WasmIntrinsics.xml</_ExtraTrimmerArgs>
+    <_ExtraTrimmerArgs Condition="'$(WasmEnableSIMD)' != 'true'">$(_ExtraTrimmerArgs) --substitutions $(MSBuildThisFileDirectory)/ILLink.Substitutions.NoWasmIntrinsics.xml</_ExtraTrimmerArgs>
```

Comment:
This is using private properties from the ILLink sdk. Since this file is shipped, we need to use an alternate way to pass these arguments.

Reply:
This property is suggested in the repo docs: https://github.com/dotnet/linker/blob/2db841fc5f6bc7592b66ffe3bd0e1c888d022f93/docs/illink-options.md?plain=1#L3-L7

```
The illink is IL linker version shipping with .NET Core or .NET 5 platforms. It's bundled with the .NET SDK and most of the options are accessible using msbuild properties but any option can also be passed using _ExtraTrimmerArgs property.
```

And it is being used by multiple other projects like xamarin-macios.
@marek-safar Is there an alternate property that should be used for passing arbitrary arguments to the linker?

Reply:
I asked @vitek-karas about it yesterday and he thinks there's currently no other way to set that.

Reply:
Adding @sbomer who knows best.

If this is an important scenario we could add more official support for it in the linker SDK bits.